### PR TITLE
fix: upgrade datatables to bootstrap 4 theme

### LINKED ISSUE #3147: UI Issue: Patient Finder previous and next button
### Describe the bug

Patient Finder previous and next button style.

### To Reproduce

1. Choose the Patient/Client
2. Click Patient
3. See the previous and next button

image

### Client configuration

- Browser: Firefox 73.0.1（64 bit）
- Operating system: Windows 10

### PR DESCRIPTION
**Pull request by stu01509**
Fixes #3147 

#### Short description of what this resolves:
Remove pt_table_next style.
Upgrade datatables to Bootstrap theme. 

image

### CODE REVIEW

**Comment by im-Amitto** [comment]
@stu01509 
Few changes that I would suggest
a. Remove space between the buttons
Screenshot from 2020-03-14 04-15-43
b. Individual search can be removed as we already have a global search field in datatable (personal opinion)
Screenshot from 2020-03-14 04-15-03
c. Sort indicator is overlapping
Screenshot from 2020-03-14 04-09-46
d. When hover over a row let's highlight the entire row instead of just the patient name.

You can find all those changes in PR #3113

File: interface/main/finder/dynamic_finder.php [hunk 1]
```diff
@@ -60,6 +60,10 @@
 <head>
     <?php Header::setupHeader(['datatables', 'datatables-colreorder', 'datatables-dt']); ?>
     <title><?php echo xlt("Patient Finder"); ?></title>
+
+
+    <link rel="stylesheet" href="./../../../public/assets/datatables.net-bs4/css/dataTables.bootstrap4.min.css">
+    <script src="./../../../public/assets/datatables.net-bs4/js/dataTables.bootstrap4.min.js"></script>
```
Comment (bradymiller) [inline interface/main/finder/dynamic_finder.php]: remove above 2 lines and instead change line 61 to:
```
<?php Header::setupHeader(['datatables', 'datatables-colreorder', 'datatables-dt', 'datatables-bs']); ?>
```
In regards to that line, here is where the magic is happening:
https://github.com/openemr/openemr/blob/master/config/config.yaml#L111-L114

**Comment by bradymiller** [comment]
hi @stu01509 and @im-Amitto ,
Agree with a,c,d above. Wouldn't pursue b since that is a very customizable feature (can set what shows up there in Administration->lists) and the global search likely will not work in many specific custom cases; doing this would require more input from community and likely some code improvements.

**Comment by stu01509** [comment]
Hi @bradymiller, @im-Amitto 

Thanks for your review, I fixed it.

**Comment by bradymiller** [comment]
hi @stu01509 , Testing well and code looks good, so bringing into codebase. Thanks for the improvements! -brady

**Pull request merged**